charts/hybrid-ingestion-runner/Chart.yaml

```diff
@@ -24,3 +24,10 @@ maintainers:
   - name: OpenMetadata
     email: support@open-metadata.org
 icon: https://open-metadata.org/assets/favicon.png
+dependencies:
+  - name: argo-workflows
+    version: "0.45.26"
+    repository: "https://argoproj.github.io/argo-helm"
+    condition: argo-workflows.enabled
```

Comment:
Can we merge with `config.argoWorkflows.enabled`? If we can get it in a single place, the better!